powerVCF.psm1

```diff
@@ -907,6 +907,146 @@ Function Remove-VCFNetworkPool {
+    This example shows how to get a list of details for a specific network associated to the network pool using ids	
+#>
+	
+	param (
+        [Parameter (Mandatory=$true)]
+            [ValidateNotNullOrEmpty()]
+            [string]$id,
+        [Parameter (Mandatory=$false)]
+            [ValidateNotNullOrEmpty()]
+            [string]$networkid
+    )
+
+    $headers = @{"Accept" = "application/json"}
+    $headers.Add("Authorization", "Basic $base64AuthInfo")
+    if ($PsBoundParameters.ContainsKey("id")) {
+        $uri = "https://$sddcManager/v1/network-pools/$id/networks"
+    }
+    if ($PsBoundParameters.ContainsKey("id") -and ($PsBoundParameters.ContainsKey("networkid"))) {
+        $uri = "https://$sddcManager/v1/network-pools/$id/networks/$networkid"
+    }
+    try {
+        if ($PsBoundParameters.ContainsKey("id")) {
+            $response = Invoke-RestMethod -Method GET -URI $uri -headers $headers
+            $response.elements
+        }
+        if ($PsBoundParameters.ContainsKey("id") -and ($PsBoundParameters.ContainsKey("networkid"))) {
+            $response = Invoke-RestMethod -Method GET -URI $uri -headers $headers
+            $response
+        }
+    }
+    catch { 
+        #Get response from the exception
+        ResponseExeception
+    }
+}
+Export-ModuleMember -Function Get-VCFNetwork
+
+Function Add-VCFNetwork {
+<#
+    .SYNOPSIS
+    Add an IP Pool to the Network of a Network Pool
+
+    .DESCRIPTION
+    The Add-VCFNetwork cmdlet connects to the specified SDDC Manager and adds a new IP Pool to an existing Network within
+	a Network Pool. 
+
+    .EXAMPLE
+    PS C:\> Add-VCFNetwork -id 917bcf8f-93e8-4b84-9627-471899c05f52 -networkid c2197368-5b7c-4003-80e5-ff9d3caef795 -ipStart 192.168.110.61 -ipEnd 192.168.110.64
+    This example shows how create a new IP Pool on the existing network for a given Network Pool
+#>
+	
+	param (
+        [Parameter (Mandatory=$true)]
+            [ValidateNotNullOrEmpty()]
+            [string]$id,
+        [Parameter (Mandatory=$true)]
+            [ValidateNotNullOrEmpty()]
+            [string]$networkid,
+        [Parameter (Mandatory=$true)]
+            [ValidateNotNullOrEmpty()]
+            [string]$ipStart,
+        [Parameter (Mandatory=$true)]
+            [ValidateNotNullOrEmpty()]
+            [string]$ipEnd
+    )
+
+    $headers = @{"Accept" = "application/json"}
+    $headers.Add("Authorization", "Basic $base64AuthInfo")
+    $uri = "https://$sddcManager/v1/network-pools/$id/networks/$networkid/ip-pools"
+    $body = '{"end": "'+$ipEnd+'","start": "'+$ipStart+'"}'
+    try {
+        $response = Invoke-RestMethod -Method POST -URI $uri -headers $headers -ContentType application/json -body $body
+        $response
+    }
+    catch { 
+        #Get response from the exception
+        ResponseExeception
+    }
+}
+Export-ModuleMember -Function Add-VCFNetwork
+
+Function Remove-VCFNetwork {
```

Comment:
Again this relates to IP Pools within a network where the function name suggests a network will be removed

Reply:
Yes after I submitted it I was thinking the same, let me add IPPool to each module

Reply:
Updated